main.go

```diff
@@ -18,6 +18,7 @@ const hofer_str = `
 * @Russell
 * @annnn
 * @Smort
+* @Azure
```

Comment:
Sorry but we are hosted on fly.io, we are not looking to be multi cloud at this point in time